### PR TITLE
feat: add blast and metadata

### PR DESCRIPTION
**Pull request by yujin512**
1. add blast to json (cross margined)
2. add metadata for blast and ondo

### CODE REVIEW

**Review by jaredvu — APPROVED** [review]

File: src/components/AssetIcon.tsx [hunk 1]
```diff
@@ -559,7 +560,6 @@ const assetIcons = {
   TWT: '/currencies/twt.png',
   UFT: '/currencies/uft.png',
   UMA: '/currencies/uma.png',
-  UNAVAILABLE: '/currencies/unavailable.png',
```
Comment (tyleroooo) [inline src/components/AssetIcon.tsx:562]: is this intended?

**Review by tyleroooo — COMMENTED** [review]
why are there 21,000 lines changed in the marketData json?

File: src/components/AssetIcon.tsx (hunk 1 above)
Comment (jaredvu) [reply src/components/AssetIcon.tsx:562]: Yeah, it was probably the linter that deleted this key.

Comment (yujin512) [reply src/components/AssetIcon.tsx:562]: I deleted because it's from my script not handling error case. is this used as a placeholder though?

**Comment by yujin512** [comment]
1. markets are sorted by cmc rank which can change from run to run (based on market cap and volume)
2. reference price changes for every market
3. some markets are added/deleted based on relevance and eligibility

**Pull request merged**